coc.md

```diff
@@ -0,0 +1,78 @@
+
+Microaggressions are brief and commonplace daily verbal, behavioral, or environmental indignities, whether intentional or unintentional, that communicate hostile, derogatory, or negative racial or sexual slights and insults. We consider microaggressions to be a form of harassment, and we will take appropriate action for incidences of microagressions.
+
+## Consequences of Unacceptable Behavior
+
+Unacceptable behavior from any community member, including sponsors and those with decision-making authority, will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event). Individuals who have been banned are not allowed to attend NodeBots SF events or participate in online discussions on the NodeBots SF meetup.com page.
+
+## If You Witness or Are Subject to Unacceptable Behavior
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. Full contact information is listed in the Contact Info section of this document. All communications will be kept strictly confidential, unless otherwise required by law. No issue will be considered too inconsequential or unimportant.
+
+Additionally, community organizers are available to help community members engage with local law enforcement or to otherwise help those experiencing unacceptable behavior feel safe. In the context of in-person events, organizers will also provide escorts as desired by the person experiencing distress.
+
+## Governing Policy and Addressing Grievances
+
+If you receive a temporary or permanent ban, your name and meetup.com photo will be added to a private ban list. You will also be prevented from RSVP'ing to future NodeBots SF events on the meetup.com website. If you receive a temporary ban, you will be notified via meetup.com private message once the ban has been lifted and you are allowed to rejoin the community.
+
+If you have been banned and show up at an event, you will be escorted off the premises by community organizers. Local law enforcement may be contacted, at the organizers' discretion.
+
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify us with a concise description of your grievance at the email address listed under Contact Info. Your grievance will be handled in accordance with our governing policies.
+
+## Contact Info
+
+If you need to report an incident, please call or text us at (415) 944-4941, email us at incident@nodebotssf.org, or talk to any of the organizers at an event. In addition, you can contact any of the following co-organizers directly:
+
+Emily Rose
+[Insert contact info here]
```

Comment:
@nexxy, can I get a public email address for you? Also, are you still on twitter, I couldn't seem to find you.